scrapers/ClubElegantAngel.yml

```diff
@@ -0,0 +1,76 @@
+name: "ClubElegantAngel"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - clubelegantangel.com/video/
+    scraper: sceneScraper
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - clubelegantangel.com/dvd/
+    scraper: movieScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Studio:
+        Name: 
+          fixed: Elegant Angel
+        URL: 
+          fixed: https://www.clubelegantangel.com
+      URL: //head/link[@rel="canonical"]/@href
+      Movies:
+        Name: //div[contains(.,"DVD :")]/a/text()
+        URL: 
+          selector: //div[contains(.,"DVD :")]/a/@href
+          postProcess:
+            - replace:
+                - regex: ^
+                  with: "https://www.clubelegantangel.com"
+      Title: //head/title/text()
+      Date:
+        selector: //div[@class="wrpHighlightUpdated"]/text()
+        postProcess:
+          - parseDate: 2006-01-02 #2012-02-18
+      #Details:
+        #selector: 
```

Comment:
can be removed, havent found description anywhere

Reply:
removed it